app/queries/user_analytics_query.rb

```diff
@@ -3,33 +3,34 @@ class UserAnalyticsQuery
   include MonthHelper
   include PeriodHelper
 
-  attr_reader :current_facility, :days_ago, :months_ago
-
-  def initialize(facility, days_ago: 30, months_ago: 6)
+  def initialize(facility, days_ago: 30, months_ago: 6, fetch_until: Date.current)
```

Comment:
The default for this in the controller is `Time.current`, but the default here is `Date.current`. Should this be a Date or a Time? Can we consistent-ize across both places?